src/common/OutputDataSocket.cc

```diff
@@ -396,6 +396,8 @@ void OutputDataSocket::append_output(ceph::buffer::list& bl)
       skipped = 1;
     } else
       ++skipped;
+
+    cond.notify_all();
```

Comment:
@yushu20171007 this is certainly ok to do, but I'm not sure what it fixes.  The only wait on cond appears to be at l.284, where we have wait(1)--I assume this amounts to 1s?  If that expression results in waiting for a "long period," then this makes sense.
*HOWEVER*, there's another error in this code, and that is:  this wait (l.284) is being done with the lock held, and is supposed to instead have 1) used unique_lock to take m_lock; 2) used cond.wait_for(l, 1s) --so that path drops the lock while waiting.  Otherwise, there is no overlap.

Reply:
1s == std::chrono::seconds(1) ;  either is ok, the point is, we should be using explicit time units from std::chrono (or aliases)

Reply:
good catch on all of this